core/store/Cargo.toml

```diff
@@ -7,13 +7,12 @@ edition = "2018"
 [dependencies]
 byteorder = "1.2"
 elastic-array = { version = "0.10" }
-kvdb = "0.1"
-kvdb-memorydb = "0.1"
-kvdb-rocksdb = "0.1.3"
+rocksdb = "0.13"
 serde = "1.0"
 serde_derive = "1.0"
 cached = "0.11.0"
 log = "0.4"
+parking_lot = "0.9"
```

Comment:
Sounds good, parking_lot also doesn't properly document when lock will fail (std does). It was std:rwlock until i was blocked the above lifetime issue and tries what parity used. Since that's resolved it's good to turn it back.